conf/webpack.config.js

```diff
@@ -138,6 +139,12 @@ function getConfig(isReactExternalized) {
                     flags: 'g',
                 },
             },
+            {
+                test: /\.(js|mjs)$/,
+                loader: 'babel-loader',
+                include: /(node_modules)/,
+                exclude: excludeFromBabel,
+            },
```

Comment:
It looks like other internal projects pass some additional configuration. Is that necessary here?

Reply:
Maybe we should just copy most of https://github.com/facebook/create-react-app/blob/3efcdbf328cff893810a9ed88a4cf4cf838484f5/packages/react-scripts/config/webpack.config.js#L394-L431

Reply:
Or is the goal, that it uses our babelrc, by not providing babelrc: false?
I am not seeing those options listed here https://github.com/babel/babel-loader

Reply:
You need to specify `babelrc: false` and `configFile: false` so that the packages are compiled according to the config we control instead of babel settings defined in the package.

It should look like the [link @priyajeet provided](https://github.com/facebook/create-react-app/blob/3efcdbf328cff893810a9ed88a4cf4cf838484f5/packages/react-scripts/config/webpack.config.js#L394-L431). If you don't want to install `babel-preset-react-app/dependencies` you can inline the necessary configuration from the file: 

https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/dependencies.js

```js
// webpack.config.js
// ...rules...
  {
    test: /\.(js|mjs)$/,
    include: /node_modules/,
    exclude: excludeFromBabel,
    loader: require.resolve('babel-loader'),
    options: {
      babelrc: false,
      cacheCompression: false,
      cacheDirectory: true,
      compact: false,
      configFile: false,
      presets: [
        [
          // 🔻install babel-preset-react-app to use this 🔻
          require.resolve('babel-preset-react-app/dependencies'),
          { helpers: true }
        ]
      ],
      sourceMaps: false,
    },
  },
```